tests/test_pagination.py

```diff
@@ -30,59 +30,64 @@ def test_run(self):
 
         incremental_streams = {key for key, value in self.expected_replication_method().items()
                                if value == self.INCREMENTAL}
-        untested_streams = self.child_streams().union({
+        direct_streams = self.child_streams().union({
+            # Data is generated automatically for 'balance_transactions' when 'charges' is created
             'balance_transactions',
             # 'charges',
             # 'coupons',
             # 'customers',
-            'disputes',
+            # 'disputes',
             # 'invoice_items',
+            # Data is generated automatically for 'invoice_line_items' when 'invoice_items' is created
             'invoice_line_items',
             # 'invoices',
             'payout_transactions',
```

Comment:
Put comment after this that this is the substream and it's not possible to solve for substreams.

Reply:
Also, attach the card number in the comment.